modules/space/translation/horizonstranslation.cpp

```diff
@@ -83,7 +79,36 @@ HorizonsTranslation::HorizonsTranslation(const ghoul::Dictionary& dictionary)
     : HorizonsTranslation()
 {
     const Parameters p = codegen::bake<Parameters>(dictionary);
-    _horizonsTextFile = absPath(p.horizonsTextFile).string();
+
+    if (std::holds_alternative<std::string>(p.horizonsTextFile)) {
+        std::string file = std::get<std::string>(p.horizonsTextFile);
+        if (!std::filesystem::is_regular_file(absPath(file))) {
+            LWARNING(fmt::format("The Horizons text file '{}' could not be found", file));
+            return;
+        }
+
+        std::vector<std::string> files;
+        files.reserve(1);
```

Comment:
No need to reserve one,  doesn't optimize anything;   either the vector already has space for at least one object (-> the reserve is a no-op) or it doesn't have space (-> the `push_back` will allocate that space).
Using reserve is useful if  you know ahead of time that you want to add a *lot* of items that could cause the vector to be reseated because the preallocated memory is not enough